src/features/search/KennisartikelDetail.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <article>
+    <utrecht-heading model-value :level="level" class="heading">
+      {{ title }}
+    </utrecht-heading>
+    <nav>
+      <ul>
+        <li
+          v-for="({ isActive, label, id }, i) in mappedSections"
+          :key="id + 'nav'"
+          :class="{ 'is-active': isActive }"
+        >
+          <span v-if="isActive">{{ label }}</span>
+          <a v-else :href="`#${id}`" @click.prevent="currentSectionIndex = i">{{
+            label
+          }}</a>
+        </li>
+      </ul>
+    </nav>
+    <section
+      v-for="{ id, html, isActive, label } in mappedSections"
+      :key="id + 'text'"
+      :class="{ 'is-active': isActive }"
+      :id="id"
+    >
+      <utrecht-heading model-value :level="level + 1">{{
+        label
+      }}</utrecht-heading>
+      <div v-html="html"></div>
+    </section>
+  </article>
+</template>
+<script setup lang="ts">
+import { cleanHtml, unEscapeHtml } from "@/helpers/html";
+import { UtrechtHeading } from "@utrecht/web-component-library-vue";
+import { nanoid } from "nanoid";
+import { computed, ref, watch } from "vue";
+
+const language = "nl";
+const knownSections = {
+  specifiekeTekst: "Inleiding",
+  procedureBeschrijving: "Aanvraag",
+  conditions: "Voorwaarden",
+  bewijs: "Bewijs",
+  kostenEnBetaalmethoden: "Kosten",
+  uitersteTermijn: "Termijn",
+  bezwaarEnBeroep: "Bezwaar",
+  vereisten: "Vereisten",
+  notice: "Bijzonderheden",
+  wtdBijGeenReactie: "Geen reactie",
+  contact: "Contact",
+  deskMemo: "KCC",
+} as const;
+
+const uniqueId = nanoid();
+
+const props = defineProps<{
+  jsonObject: any;
```

Comment:
We could maybe make this more descriptive, `jsonObject` doesn't tell anything about the expected object. I understand that we might not yet be sure _what_ the object will contain (due to it being `any`).

However, maybe something like `kennisartikelRawObject` (working title 😄 )

Reply:
Done